crowbar_framework/app/assets/javascripts/jquery/nodeList.js

```diff
@@ -317,6 +321,40 @@
             break;
         }
       }
+
+      if (constraints.platform !== undefined && constraints.platform && !cluster) {
```

Comment:
No need for the cluster check as it doesn't have a platform constraint attribute.